card/mod/03_machines/set/type/coffee_script.rb

```diff
@@ -18,7 +18,7 @@ def compile_coffee script
   Uglifier.compile( compile_coffee format(:js)._render_raw )
 end
 
-store_machine_output :filetype => "js"
+store_machine_output filetype: "js"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.